regl-camera.js

```diff
@@ -60,10 +69,16 @@ function createCamera (regl, props_) {
     return Math.min(Math.max(x, lo), hi)
   }
 
-  function updateCamera () {
+  function updateCamera (props) {
+    Object.keys(props).forEach(function (prop) {
+      cameraState[prop] = props[prop]
```

Comment:
Not sure of the desirability of this but I really want to be able to drive camera movement without using mouse interaction e.g. `camera({ dtheta: 0.1 }, () => {})`.